README.adoc

```diff
@@ -128,11 +134,18 @@ mvn liberty:stop-server
 
 Navigate to the `start` directory to begin.
 
-The MicroProfile Fault Tolerance API was added as a dependency to your [hotspot file=4]`pom.xml` file. Look for
-the dependency with the [hotspot=73-78 file=4]`mpFaultTolerance` artifact ID. Adding this dependency allows you to use the
-fault tolerance policies in your microservices.
+Start Open Liberty in development mode, which starts the Open Liberty server and listens for file changes:
+
+[role=command]
+```
+mvn liberty:dev
+```
+
+The MicroProfile Fault Tolerance API is included in the MicroProfile dependency specified by your [hotspot file=4]`pom.xml` file. 
```

Comment:
Change to:
"The MicroProfile Fault Tolerance API is included in the MicroProfile dependency that is specified in your `pom.xml` file."

Reply:
Updated